ci/experimental/docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# We use manylinux1 base image because pyarrow_manylinux2010 has a bug and wheel failed to be audited
```

Comment:
I think this file is the same as `ci/experimental/docker/connector_build/Dockerfile` I think we don't need this